### PR TITLE
snowflake native apps: updating docs w/ scheduled syncs and some cleanup ahead of public preview

### PR DESCRIPTION
**Pull request by brianjlai**
## What

Just tidying up some of our docs ahead of the release. I adjusted some of the auth commands for consistency and the biggest change was additional documentation now that it supports scheduled syncs. + a few changes suggested by Kat and Alex B when he was testing out the app

### CODE REVIEW

File: docs/snowflake-native-apps/linkedin-ads.md [hunk 1]
```diff
@@ -37,23 +40,23 @@ In order for the LinkedIn Ads Connector by Airbyte to query LinkedIn, you will n
 ## Snowflake Native App Authorizations
 
 :::note
-By default the app will be installed using the name `AIRBYTE_LINKEDIN_ADS`, but if you renamed the app during installation, you will have to use that name as a reference.
```
Comment (brianjlai) [inline docs/snowflake-native-apps/linkedin-ads.md:40]: I already changed the name of this in our provider account to be consistent with Facebook

File: docs/snowflake-native-apps/facebook-marketing.md [hunk 2]
```diff
@@ -21,8 +21,11 @@ Do not refresh the Apps page while the application is being installed. This may
 2. On the left sidebar, click `Marketplace`.
 3. Search for `Facebook Marketing Connector` by Airbyte or navigate to https://app.snowflake.com/marketplace/listing/GZTYZ9BCRT8/airbyte-facebook-marketing-connector-by-airbyte
 4. Click `Request`. This will send a request that we will manually service as soon as we can.
+
+![](./facebook-marketing-shared-application.png)
+
 5. On the left sidebar, click `Apps`.
-6. Under the `Recently Shared with You` section, you should see the `Facebook Marketing Connector by Airbyte`. Click `Get`.
+6. Under the `Recently Shared with You` section, you should see the `Airbyte Facebook Marketing Connector` by Airbyte. Click `Get`.
```
Comment (brianjlai) [inline docs/snowflake-native-apps/facebook-marketing.md:28]: This process could change depending on what the process becomes when we start using the marketplace listing

File: docs/snowflake-native-apps/facebook-marketing.md [hunk 3]
```diff
@@ -161,25 +170,44 @@ The table where the schema will be saved. Snowflake's table [naming convention](
...
+
+### Scheduled Syncs
+While creating a connection, you can specify a "Replication Frequency" which will dictate how often your data will be extracted from
+Facebook Marketing and loaded into your Snowflake database. This process is started automatically according to your schedule and does not
+require that you manually trigger syncs. For example, if you create a connection at 10:15 AM and set your replication frequency to
+hourly, then a sync will be started immediately. The next sync will start at 11:15 AM onwards. Only one sync is active at the same
+time. In the event that your sync runs longer than one hour, a new sync will start at the next available time.
+
+### Manual Syncs
+In addition to scheduled syncs, you can also configure a connection only sync data on-demand by setting "Replication Frequency" to
```
Comment (clnoll) [inline docs/snowflake-native-apps/facebook-marketing.md:200]:
```suggestion
In addition to scheduled syncs, you can also configure a connection to only sync data on-demand by setting "Replication Frequency" to
```

File: docs/snowflake-native-apps/facebook-marketing.md [hunk 4]
```diff
@@ -161,25 +170,44 @@ The table where the schema will be saved. Snowflake's table [naming convention](
...
+`MANUAL`. After creating a connection, from the `Connections List` page, you can use the "Sync Now" button to trigger a sync of
+your API data to your Snowflake database. You can also use this button to manually trigger connections that sync according to a
+schedule. If there is already a sync in progress, this button will be greyed out.
```
Comment (clnoll) [inline docs/snowflake-native-apps/facebook-marketing.md:203]:
```suggestion
schedule. If there is already a sync in progress, this button will be disabled. 
```

File: docs/snowflake-native-apps/linkedin-ads.md [hunk 5]
```diff
@@ -136,25 +137,44 @@ The table where the schema will be saved. Snowflake's table [naming convention](
...
+
+### Scheduled Syncs
+While creating a connection, you can specify a "Replication Frequency" which will dictate how often your data will be extracted from
+LinkedIn Ads and loaded into your Snowflake database. This process is started automatically according to your schedule and does not
+require that you manually trigger syncs. For example, if you create a connection at 10:15 AM and set your replication frequency to
+hourly, then a sync will be started immediately. The next sync will start at 11:15 AM onwards. Only one sync is active at the same
+time. In the event that your sync runs longer than one hour, a new sync will start at the next available time.
+
+### Manual Syncs
+In addition to scheduled syncs, you can also configure a connection only sync data on-demand by setting "Replication Frequency" to
```
Comment (clnoll) [inline docs/snowflake-native-apps/linkedin-ads.md:167]:
```suggestion
In addition to scheduled syncs, you can also configure a connection to only sync data on-demand by setting "Replication Frequency" to
```

File: docs/snowflake-native-apps/linkedin-ads.md [hunk 6]
```diff
@@ -136,25 +137,44 @@ The table where the schema will be saved. Snowflake's table [naming convention](
...
+`MANUAL`. After creating a connection, from the `Connections List` page, you can use the "Sync Now" button to trigger a sync of
+your API data to your Snowflake database. You can also use this button to manually trigger connections that sync according to a
+schedule. If there is already a sync in progress, this button will be greyed out.
```
Comment (clnoll) [inline docs/snowflake-native-apps/linkedin-ads.md:170]:
```suggestion
schedule. If there is already a sync in progress, this button will be disabled. 
```

**Review by clnoll — APPROVED** [review]
Looks great @brianjlai! Just a couple of little wording suggestions.

**Pull request merged**